### ISSUE TITLE
V7 An error occurred when importModels was used in windows ` Only URLs with a scheme in: file, data are supported by the default ESM loader.`

### ISSUE THREAD

**Issue by yuntian001** [sequelize/sequelize#16313]
## Issue Creation Checklist

- [x] I understand that my issue will be automatically closed if I don't fill in the requested information
- [x] I have read the [contribution guidelines](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)

## Bug Description
Sequelize  V7
I got an error when I used importModels to match a global schema file in windows
```
   Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid f
ile:// URLs. Received protocol 'd:'  [stack]:Error: Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:
// URLs. Received protocol 'd:'
    at new NodeError (node:internal/errors:387:5)
    at throwIfUnsupportedURLScheme (node:internal/modules/esm/resolve:1017:11)
    at defaultResolve (node:internal/modules/esm/resolve:1097:3)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:837:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ESMLoader.import (node:internal/modules/esm/loader:521:22)
    at importModuleDynamically (node:internal/modules/cjs/loader:1094:29)
    at importModuleDynamicallyWrapper (node:internal/vm/module:438:21)
    at importModuleDynamically (node:vm:389:46)
```
### Reproducible Example
```
import { importModels } from '@sequelize/core';

new Sequelize({
    dialect: 'mysql',
    database:  'test', //数据库名称
    host: '127.0.0.1',
    port: '3306',
    username: 'root',
    password: 'root',
    timezone: '+08:00', //时区
    benchmark: true,
    models: await importModels((__dirname + '/../app/**/*.entity.{ts,js}').replace(/\\/g,'/')),
 });
```
### What do you expect to happen?
no error

### What is actually happening?

```
   Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid f
ile:// URLs. Received protocol 'd:'  [stack]:Error: Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:
// URLs. Received protocol 'd:'
    at new NodeError (node:internal/errors:387:5)
    at throwIfUnsupportedURLScheme (node:internal/modules/esm/resolve:1017:11)
    at defaultResolve (node:internal/modules/esm/resolve:1097:3)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:837:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ESMLoader.import (node:internal/modules/esm/loader:521:22)
    at importModuleDynamically (node:internal/modules/cjs/loader:1094:29)
    at importModuleDynamicallyWrapper (node:internal/vm/module:438:21)
    at importModuleDynamically (node:vm:389:46)
```
### Environment

Please answer the questions below. If you don't, your issue will be closed.

- Sequelize version: 7.0.0-alpha.27
- Node.js version: 16.18.1
- If TypeScript related: TypeScript version: 5.1.3
- Database & Version: mysql 5.7
- Connector library & Version: mysql2:3.4.1

## Would you be willing to resolve this issue by submitting a Pull Request?

- [x] Yes, I have the time and I know how to start.
- [ ] Yes, I have the time but I will need guidance.
- [ ] No, I don't have the time, but my company or I are [supporting Sequelize through donations on OpenCollective](https://opencollective.com/sequelize).
- [ ] No, I don't have the time, and I understand that I will need to wait until someone from the community or maintainers is interested in resolving my issue.

---

*Indicate your interest in the resolution of this issue by adding the 👍 reaction. Comments such as "+1" will be removed.*

**Comment by WikiRik** [comment]
This will likely be fixed in #16283

**Comment by yuntian001** [comment]
> This will likely be fixed in #16283

yes #16283 could solve my problem

**Comment by yuntian001** [comment]
> This will likely be fixed in #16283

What is the date of the V7 live fix plan
